pkg/logs/status/status.go

```diff
@@ -6,6 +6,8 @@
 package status
 
 import (
+	"expvar"
+	"github.com/DataDog/datadog-agent/pkg/logs/metrics"
```

Comment:
nits, can you move this at the very end of the list of dependencies ?

Reply:
done